spec/spec_helper.rb

```diff
@@ -29,4 +29,7 @@
   # automatically. This will be the default behavior in future versions of
   # rspec-rails.
   config.infer_base_class_for_anonymous_controllers = false
+
+  config.include Capybara::DSL
```

Comment:
This would be better in `spec/support/capybara.rb`.  It should also be included by default in all request/feature specs.  It may actually just need a `require 'capybara/rspec'`, and maybe a `require 'capybara/rails'`.